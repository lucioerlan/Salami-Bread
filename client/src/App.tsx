import React from 'react';

import './App.css';
import logo from './logo.svg';

function App() {
  return (
    <div className="App" data-testid="app">
      <header className="App-header"></header>
    </div>
  );
}

export default App;
