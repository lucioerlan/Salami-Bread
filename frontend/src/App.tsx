import { useRoutes } from 'react-router-dom';
import routes from './routes';

const App = () => {
  const routing = useRoutes(routes('dsd'));

  return (
    <div className="App" data-testid="app">
      {routing}
    </div>
  );
};

export default App;
