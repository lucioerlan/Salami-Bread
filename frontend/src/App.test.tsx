import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('react-router-dom', () => ({
  useRoutes: () => jest.fn(),
}));

describe('App', () => {
  let rtlContainer: HTMLElement;

  beforeEach(() => {
    const { container } = render(<App />);
    rtlContainer = container;
  });

  it('should matches with snapshot', () => {
    expect(rtlContainer.firstChild).toMatchSnapshot();
  });
});
