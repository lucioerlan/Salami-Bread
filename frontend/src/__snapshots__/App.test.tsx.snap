// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should matches with snapshot 1`] = `
<div
  class="App"
  data-testid="app"
/>
`;
